spec/models/user_spec.rb

```diff
@@ -0,0 +1,5 @@
+require 'rails_helper'
+
+RSpec.describe User, type: :model do
+  pending "add some examples to (or delete) #{__FILE__}"
+end
```

Comment:
In order for me to merge this, you'll need to add test for your validations here. For example,

```
  it { should have_valid(:first_name).when('Jon', 'Sam')}
  it { should_not have_valid(:first_name).when(nil, '')}

  it { should have_valid(:last_name).when('Smith', 'Jackson')}
  it { should_not have_valid(:last_name).when(nil, '')}
```